README.md

```diff
@@ -29,6 +30,7 @@ Export your GitHub API token:
     export GITHUB_OMNI_AUTH_KEY="github-application-key";
     export GITHUB_CALLBACK_URL="http://<your-url-and-port>";  // must match that given to Github! Would be 127.0.0.1:port for testing
 
+    If authentication is 'turned on', organization 'team' members who are collaborators and exclusively on a 'pull' only team will be able to see a repo, however the deploy button will be diasbled for them.
```

Comment:
does this mean that collaborators have the deploy button disabled for them?

if it is only disabled for members on a `pull only`, perhaps we mean
```
If authentication is 'turned on', organization 'team' members who are collaborators and exclusively on a 'pull' only team will be able to see a repo. However the deploy button will be disabled for members who are not collaborators or owners of the repo.
```